cmake/Modules/RpcExternal.cmake

```diff
@@ -0,0 +1,58 @@
+set(GRPC_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/external/grpc")
```

Comment:
Should this file named `GrpcExternal.cmake`?

Reply:
Done